numpy_challenge.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+
+if __name__ == "__main__":
+    arr = np.eye(42)
+    arrr = np.array(((10, 9, 22), (1, 4, 7)))
+    arrrr = arrr[arrr < 18]
+    arrrrrr = np.arange(1, 15, 7)
+    fast = arrr.reshape(3, -1)
+
+
+def matrix_multiplication(arg1, arg2):
+    mult = np.matmul(arg1, arg2)
+    return mult
+
+
+def multiplication_check(your_list):
+    result = True
+    for elem in range(len(your_list) - 1):
+        if your_list[elem].shape[1] == your_list[elem + 1].shape[0]:
+            continue
+        else:
+            result = False
+            break
+    return result

```

Comment:
Нейминг переменных не очень. В этой и других функциях
```suggestion
def multiplication_check(matrices_list):
    result = True
    for idx in range(len(your_list) - 1):
        if matrices_list[idx].shape[1] != matrices_list[idx + 1].shape[0]:
            return False
    return True
```